### PR TITLE
Rename occurrence search sorting options

### PR DESCRIPTION
**Pull request by raivop**
[#161382593]

### CODE REVIEW

File: grails-app/i18n/messages_et.properties [hunk 1]
```diff
@@ -355,7 +355,7 @@ recordcore.iao.01.02 = inferred associated occurrences
 recordcore.iao.02 = The occurrence is associated with a representative record
 recordcore.iao.03 = For more information see
 recordcore.iao.04 = Inferred associated occurrence details
-recordcore.occurenceevent.title = Sündmus
+recordcore.occurrencevent.title = Sündmus
```
Comment (filippivanov) [inline grails-app/i18n/messages_et.properties]: From one typo to another? :)

Comment (raivop) [reply grails-app/i18n/messages_et.properties]: :(

Comment (timo-piirmann) [reply grails-app/i18n/messages_et.properties]: But is it ready to be merged?

**Comment by raivop** [comment]
It's now ready to be merged. Please squash.

**Pull request merged**